### PR TITLE
fix(association-select): no upperCase html attibutes

### LINKED ISSUE #202: association-select ignores placeholder value
In rc7 of aurelia-orm, the association-select ignores a custom placeholder value. i have set following properties, but in the final html code always '- select a value -' gets rendered as placeholder.

`
 <association-select property="name"
                                    value.bind="currentStudioId"
                                    repository.bind="studioRepo"
                                    selectablePlaceholder="true"
                                    criteria.bind='{where:{isAvailable:{"==":true}}}'
                                    placeholderText="Select Studio">
                </association-select>
`

### PR DESCRIPTION
**Pull request by doktordirk**
closes https://github.com/SpoonX/aurelia-orm/issues/202

### CODE REVIEW

**Comment by doktordirk** [comment]
what about the 'manyAssociation' attribute?

**Comment by RWOverdijk** [comment]
What about it?

Anyway, as to this solution, I think it's best to do this elsewhere. For starters this shouldn't be called to early (when does that happen to begin with?) if it does, perhaps throw an error indicating you're loading stuff too early.

**Comment by doktordirk** [comment]
the thing is, there are no upperCase html attributes in the first place. that's the specs for html

File: doc/components.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ Get all entries of the view-models 'userRepository' repository property and popu
   association.bind="[data.page, data.group]"
   manyAssociation.bind="data.category"
   criteria.bind='{where:{age:{">":18}}}'
-  selectablePlaceholder="true"
-  placeholderText="- Any -"
+  selectableplaceholder="true"
```
Comment (RWOverdijk) [inline doc/components.md:59]: Use dash separator in stead. `selectable-placeholder`

File: src/component/association-select.js [hunk 2]
```diff
@@ -112,6 +112,11 @@ export class AssociationSelect {
    * @return {Promise}
    */
   buildFind() {
+    // resolve if called too early
+    if (!this.repository) {
```
Comment (RWOverdijk) [inline src/component/association-select.js:116]: If this happens, the developer did something weird. Log a warning in stead of silently resolving.

**Comment by doktordirk** [comment]
this needs rethinking

**Pull request closed without merging**